dask_ml/utils.py

```diff
@@ -241,6 +248,44 @@ def _format_bytes(n):
     return '%d B' % n
 
 
+@contextlib.contextmanager
+def _timer(name, *args, _logger=None, level='info', **kwargs):
+    """
+    Output execution time of a function to the given logger level
+
+    :param str name: How to name the timer (will be in the logs)
```

Comment:
We use [numpydoc](http://numpydoc.readthedocs.io/en/latest/format.html) style docstrings. Could you update these to use a `Parameters` section (don't need ad `Returns`).